README.md

```diff
@@ -556,24 +556,7 @@ applications on your system. Configure your terminal to use this font:
 - **Guake**: Right Click on an open terminal and open *Preferences*. Under *Appearance* 
   tab, uncheck *Use the system fixed width font* (if not already) and select `MesloLGS NF Regular`. 
   Exit the Preferences dialog by clicking *Close*.  
-- **Alacritty**: Config is consumed from one of these [four locations](https://github.com/alacritty/alacritty#configuration). If it does not exist, copy `alacritty.yaml` from [Alacritty releases](https://github.com/alacritty/alacritty/releases) to `~/.config/alacritty/`. Locate `Font Configuration` section @ line 92. v5.0:
-  ```yaml
-  font:
-    normal:
-      family: MesloLGS NF
-      style: Regular
-    bold:
-      family: MesloLGS NF
-      style: Bold
-    italic:
-      family: MesloLGS NF
-      style: Italic
-    bold_italic:
-      family: MesloLGS NF
-      style: Bold Italic
-
-    size: 11.0
-  ```
+- **Alacritty**: If you have not already generated a config file, copy `alacritty.yaml` from [Alacritty releases](https://github.com/alacritty/alacritty/releases) to `~/.config/alacritty/`. Change `family` to `MesloLGS NF` by following [these](https://github.com/alacritty/alacritty/wiki/Changing-the-default-font) instructions.
```

Comment:
The instructions must specify how to set Alacritty font to MesloLGS NF. This PR isn't quite there.

The first sentence seems confusing (why generated? why copy?) and inaccurate (Alacritty looks for the config in 4 places and the suggested file is neither the first nor the last of them). The second sentence links to a wiki that shows how to set font to "Droid Sans Mono Slashed for Powerline".

Reply:
It is definitely confusing. The multiple locations Alacritty reads from are based on the XDG base directory spec but the project's own wiki entry for font config only mentions one specific directory. $XDG_CONFIG_HOME being equal to ~/.config/ in the spec. d369295 simplifies much more.